Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -127,18 +138,18 @@ class OpenPdfInHelper: NSObject, OpenInHelper, UIDocumentInteractionControllerDe
     fileprivate var filepath: URL?
 
     required init?(response: URLResponse) {
-        guard let MIMEType = response.mimeType, MIMEType == MimeType.PDF.rawValue && UIApplication.shared.canOpenURL(URL(string: "itms-books:")!),
+        guard let MIMEType = response.mimeType, MimeType(rawValue: MIMEType) != nil && UIApplication.shared.canOpenURL(URL(string: "itms-books:")!),
             let responseURL = response.url else { return nil }
         url = responseURL
         super.init()
-        setFilePath(response.suggestedFilename ?? url.lastPathComponent )
+        setFilePath(response.suggestedFilename ?? url.lastPathComponent, mimeType: MIMEType)
     }
 
-    fileprivate func setFilePath(_ suggestedFilename: String) {
+    fileprivate func setFilePath(_ suggestedFilename: String, mimeType: String) {
         var filename = suggestedFilename
         let pathExtension = filename.asURL?.pathExtension
-        if pathExtension == nil {
```

Comment:
I think we could simplify this all into something like 
``` swift 
let filename: String
if suggestedFilename.asURL?.pathExtension == nil {
    filename = suggestedFilename + mimeType.fileExtension
} else {
    filename = suggestedFilename
}
filepath = documentDirectory.appendingPathComponent(filename)
```

what do you think? We can avoid the `var` this way.